tenable/io/v3/vm/policies/api.py

```diff
@@ -0,0 +1,395 @@
+'''
+Policies
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`policies <policies>` API.
+
+Methods available on ``tio.v3.vm.policies``:
+
+.. rst-class:: hide-signature
+.. autoclass:: PoliciesAPI
+    :members:
+'''
+from io import BytesIO
+from typing import BinaryIO, Dict, Optional, Union
+
+from requests import Response
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.base.iterators.explore_iterator import (CSVChunkIterator,
+                                                           SearchIterator)
+from tenable.io.v3.vm.policies.schema import PolicySchema
+
+# from tenable.utils import policy_settings, dict_merge
```

Comment:
We can keep as of now, with V3 API integration we can enable that part since this branch will merge with [Task/in 581 v3 standardization](https://github.com/dhvanilshah-tenable/pyTenable/pull/104/files#)